tests/conftest.py

```diff
@@ -117,5 +119,6 @@ def app(app):
         RecordResource(service=RecordService()).as_blueprint("base_resource")
     )
     app.register_blueprint(custom_bp)
+    app.register_error_handler(HTTPException, handle_http_exception)
```

Comment:
minor: like discussed this could be set on the blueprint. For tests this is fine.